examples/README.md

```diff
@@ -0,0 +1,11 @@
+Python wrapper examples
+-----------------------
+
+In this part, you can find and share real use cases for the OVH python wrapper
```

Comment:
I'd say "study" rather than "share" as this is closer to our goal here. Capitalize 'Python' ?